I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/custom/login.rpy", line 28, in script
    $say()
  File "game/custom/login.rpy", line 28, in <module>
    $say()
  File "game/script.rpy", line 212, in say
    term(desc + "\n" + append)
  File "renpy/common/00nvl_mode.rpy", line 314, in do_display
    **display_args)
IOError: Couldn't find file 'bg0.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/custom/login.rpy", line 28, in script
    $say()
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/custom/login.rpy", line 28, in <module>
    $say()
  File "game/script.rpy", line 212, in say
    term(desc + "\n" + append)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 877, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "renpy/common/00nvl_mode.rpy", line 314, in do_display
    **display_args)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 508, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/core.py", line 2526, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/display/core.py", line 2825, in interact_core
    renpy.audio.audio.interact()
  File "/Applications/renpy-6.99.12.4-sdk/renpy/audio/audio.py", line 930, in interact
    periodic()
  File "/Applications/renpy-6.99.12.4-sdk/renpy/audio/audio.py", line 864, in periodic
    c.periodic()
  File "/Applications/renpy-6.99.12.4-sdk/renpy/audio/audio.py", line 403, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/audio/audio.py", line 67, in load
    rv = renpy.loader.load(fn)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/loader.py", line 543, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'bg0.mp3'.

Darwin-16.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
> turing 0.1
